src/frontend/app/desk/package.json

```diff
@@ -9,27 +9,36 @@
     "start": "next start",
     "lint": "next lint",
     "prettier": "prettier --write .",
-    "stylelint": "stylelint \"**/*.css\""
+    "stylelint": "stylelint \"**/*.css\"",
+    "test": "jest",
+    "test:watch": "jest --watch"
   },
   "dependencies": {
     "@openfun/cunningham-react": "2.4.0",
     "next": "14.0.4",
-    "react": "18",
-    "react-dom": "18"
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
   },
   "devDependencies": {
-    "@types/node": "20",
-    "@types/react": "18",
-    "@types/react-dom": "18",
+    "@testing-library/jest-dom": "6.2.0",
+    "@testing-library/react": "14.1.2",
+    "@types/jest": "29.5.11",
+    "@types/node": "20.10.8",
+    "@types/react": "18.2.47",
+    "@types/react-dom": "18.2.18",
     "@typescript-eslint/eslint-plugin": "6.18.1",
     "eslint": "8.0.1",
     "eslint-config-next": "14.0.4",
     "eslint-config-prettier": "9.1.0",
+    "eslint-plugin-jest": "27.6.2",
     "eslint-plugin-prettier": "5.1.2",
+    "eslint-plugin-testing-library": "6.2.0",
+    "jest": "29.7.0",
+    "jest-environment-jsdom": "29.7.0",
     "prettier": "3.1.1",
     "stylelint": "16.1.0",
     "stylelint-config-standard": "36.0.0",
     "stylelint-prettier": "5.0.0",
-    "typescript": "5"
+    "typescript": "5.3.3"
```

Comment:
Actually I am not bumping, but pinning. This package was generated by next, I noticed that it was not pinning some versions, I prefer to pin them to be sure all is compatible, we will plug the renovate bot to bump the versions for us, it will be separated in different PR, we will have a good control of it.